Description: Disables the @Generated annotations by default (do not compile with Java 9+)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/processor/src/main/java/org/jboss/logging/processor/apt/LoggingToolsProcessor.java
+++ b/processor/src/main/java/org/jboss/logging/processor/apt/LoggingToolsProcessor.java
@@ -186,7 +186,7 @@
                 }
             }
         }
-        final boolean addGeneratedAnnotation = Boolean.parseBoolean(processingEnv.getOptions().getOrDefault(ADD_GENERATED_ANNOTATION, "true"));
+        final boolean addGeneratedAnnotation = Boolean.parseBoolean(processingEnv.getOptions().getOrDefault(ADD_GENERATED_ANNOTATION, "false"));
         boolean generate = true;
         final Validator validator = new Validator(processingEnv);
 
